css/index.css

```
@@ -1 +1,178 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Inter", sans-serif;
+  display: flex;
+}
 
+@font-face {
+  font-family:'Nexa Extra Light';
+  font-family:'Nexa Heavy';
+  src: url('../fonts/Nexa-ExtraLight.ttf') format('truetype');
+  src: url('../fonts/Nexa-Heavy.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+.login-information__container{
+  max-width: 1440px;
+  margin: 0 auto;
+  margin-top: 100px;
```

El diseño dice que es de 29px en movil y 32 en desktop
```suggestion
  margin-top: 29px;
```